### DISCUSSION TITLE
Is 2.3.2 release compatible with Ubuntu Server 24.04 ?

### DISCUSSION THREAD

**Discussion by Joaquim3** [modoboa/modoboa#3358] [General]
Hi,

1. Is the 2.3.2 release compatible with Ubuntu Server 24.04 ?
2. Is this project still active ? Because latest realse was on August 2022.

Kind Regards,

**Comment by Peshala84** [comment]
The 2.3.2 release's compatibility with Ubuntu Server 24.04 depends on the specific software and its dependencies. If the project hasn't had updates since August 2022, it may no longer be actively maintained

**Comment by tonioo** [comment]
Look at this repository activity before spreading wrong information.

**Reply by Joaquim3** [reply]
Thank you very much @tonioo .  😀

Looking forward to test it on Ubuntu 24.04
I'll come back here to display my results.
